code/modules/mob/living/silicon/robot/emote.dm

```diff
@@ -70,3 +83,13 @@
 			playsound(loc, 'sound/machines/buzz-two.ogg', 50, 0)
 		else
 			to_chat(src, "<span class='warning'>You can only use this emote when you're out of charge.</span>")
+
+/datum/emote/sound/silicon/yes
+	key = "yes"
+	message = "blips affirmatively."

```

Comment:
blip is a bit more... discordant...?

Beep might be better.

Reply:
But it's a blip!